src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Verify.java

```diff
@@ -87,26 +90,39 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
         while (row.hasNext()) {
           Entry<Key,Value> entry = row.next();
 
-          if (rowText == null)
+          if (rowText == null) {
             rowText = entry.getKey().getRow();
+          }
 
           long curr = Long.parseLong(entry.getKey().getColumnQualifier().toString());
 
-          if (curr - 1 != prev)
-            throw new Exception(
-                "Bad marker count " + entry.getKey() + " " + entry.getValue() + " " + prev);
+          if (curr - 1 != prev) {
+            log.error("Bad market count. Current row: {} {}, Previous row marker: {}",
+                entry.getKey(), entry.getValue(), prev);
+            errorFound = true;
+            break;
```

Comment:
As it stands, this break will be hit and break out of the inner while loop and avoid both of the next checks since `prev` will never be reassigned value to `curr` which is at the end of the inner while loop. Removing both `breaks` in the inner loop would hit both checks but unsure of many redundant errors would get logged, if any.

Reply:
@keith-turner  Any thoughts on this and what approach I should take?

Reply:
> @keith-turner Any thoughts on this and what approach I should take?

Removing all breaks seems like a good way to go, as long as what it logs is accurate I think its ok if it logs a bunch.  Its good to see all of the problems in the data.

>  It could also affect the outer if statment since prev is never assigned the value of curr which happens at the end of the inner while loop

I agree the breaks in the inner loop could cause the check following the inner loop to report a false positive.

Reply:
I will make these changes